ha/core/cluster/cluster_manager.py

```diff
@@ -304,17 +305,16 @@ def controller_list(self) -> list:
         """
         return list(self._controllers.keys())
 
+
     def get_system_health(self, element: CLUSTER_ELEMENTS = CLUSTER_ELEMENTS.CLUSTER.value, depth: int = 1, **kwargs) -> json:
         """
         Return health status for the requested elements.
-
         Args:
             element ([CLUSTER_ELEMENTS]): The element whose health status is to be returned.
             depth ([int]): A depth of elements starting from the input "element" that the health status
                 is to be returned.
             **kwargs([dict]): Variable number of arguments that are used as filters,
                 e.g. "id" of the input "element".
-
         Returns:
```

Comment:
This return description needs to be changed since we are raising exception instead of a error return value.
I know it is not in the scope of this PR but we can cover it in some upcoming PR ; So whoever is calling this does not need to do additional handling for error values

Reply:
Still we return error values, not from here but from the called classes.